KalturaWowzaServer/release/register_media_server.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+require_once('../app/clients/php5/KalturaClient.php');
+
+$kaltura_system_ini = "/etc/kaltura.d/system.ini";
+$broadcast_file = "../app/configurations/broadcast.ini";
```

Comment:
Same here, you can't trust this file existence, have your own configuration file.